ts/features/wallet/component/RemovePaymentMethod.tsx

```diff
@@ -55,10 +54,10 @@ export const RemovePaymentMethod: React.FunctionComponent<Props> = props => (
 export const useRemovePaymentMethodBottomSheet = (
   representation: PaymentMethodRepresentation
 ) => {
-  const { present, dismiss } = useBottomSheetModal();
+  const { present, dismiss } = useIOBottomSheetRaw(350);
 
-  const openModalBox = (onConfirm: () => void) => {
-    const bottomSheetProps = bottomSheetRawConfig(
+  const openBottomSheet = (onConfirm: () => void) =>
+    present(
       <RemovePaymentMethod
```

Comment:
Why the `RemovePaymentMethod` component is exported?

Reply:
I can't get it. What do you mean with `exported`?

Reply:
[Here](https://github.com/pagopa/io-app/blob/50b3ffccdc301da4beef4462881aa5eea10531d2/ts/features/wallet/component/RemovePaymentMethod.tsx#L26).
This component is only used inside this file, is `export` key needed?

Reply:
Got it. Thanks
https://github.com/pagopa/io-app/pull/2614/commits/0a9d2ede20acc3d97c4fc05f6ee94b5736c069a0